src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccessStep2.tsx

```diff
@@ -611,6 +611,7 @@ const RequestDataAccessStep2: React.FC<RequestDataAccessStep2Props> = props => {
             <UserSearchBox
               id={'requesters'}
               onSelectCallback={onSelectUserCallback}
+              filterUserIds={accessors.map(user => user.profile.ownerId)}
```

Comment:
Pass in a list of users (accessors) that's already been selected, so they don't show up in the user search box.